ugs-core/src/com/willwinder/universalgcodesender/uielements/panels/SendStatusPanel.java

```diff
@@ -196,6 +204,19 @@ private void initComponents() {
         add(remainingTimeValue);
         add(durationLabel, AL_RIGHT);
         add(durationValue);
+        add(latestCommentLabel, "span 2, wrap");
+        add(latestCommentValueLabel, "growx, span 2, wrap");
+
+        latestCommentValueLabel.setOpaque(false);
+        latestCommentValueLabel.setWrapStyleWord(true);
+        latestCommentValueLabel.setLineWrap(true);
+
+        addComponentListener(new ComponentAdapter() {
+            @Override
+            public void componentResized(ComponentEvent event) {
+               latestCommentValueLabel.setMaximumSize(new Dimension(SendStatusPanel.this.getWidth() - 10, 30));
```

Comment:
Is this to make sure the comment label is padded? There are options to do this sort of thing with MigLayout I can lookup.

Reply:
I spent quite a lot of time trying to get the last comment text to do what I wanted it to. When it was on the DRO I could get the comment text to do what I wanted, then the other components would break. I wanted the text to wrap and it proved rather challenging. If it can be done straight in MigLayout I'm happy to see that solution, I just don't think I can figure it out myself!

Reply:
Ugh, yeah Swing text wrapping is awful. I took a stab at it once with `NarrowOptionPane`, I think the recommended approach is to embed HTML (seriously).

Reply:
Marking as "won't fix" for tracking my tasks. If this is important to change, please give guidance.